_jobs/positions/__QA-testing-engineer.md

```diff
@@ -0,0 +1,54 @@
+---
+title: QA Testing & Automation Engineer
+is_fellow: true
+lever_link:
+position: 1
+lede_markdown: We’re looking for skilled manager and mentor to lead engineering practices for City of Austin’s Design, Technology, and Innovation Fellows program.
+---
+
+## Overview
+We are looking for an experienced, skilled and passionate QA Testing & Automation professional who will help streamline our QA process for current and future projects ensuring timely and frequent release schedules without compromising on quality and accessibility for Austin residents.
```

Comment:
We are looking for an experienced, skilled, and passionate QA Testing & Automation professional to help streamline our QA process for current and future projects. The person in this role will ensure timely and frequent release schedules without compromising on quality and accessibility for Austin residents.